dispatcher/linux/mfxloader.cpp

```diff
@@ -28,6 +28,9 @@ namespace MFX {
 #elif defined(__x86_64__)
     #define LIBMFXSW "libvplswref64.so.1"
     #define LIBMFXHW "libmfxhw64.so.1"
+#elif defined(__aarch64__)
```

Comment:
Per the spec: https://spec.oneapi.com/versions/latest/elements/oneVPL/source/programming_guide/VPL_prg_session.html
The name of the runtime should only change based on bitness. 

In order to support building on aarch64 instead of getting unsupported architecture, please update the condition on line 28 to include both `__x86_64__` and `__aarch64__`.

Reply:
Updated PR with requested change.